src/lib/utils/skin.ts

```diff
@@ -110,3 +111,19 @@ export function isSkin(asset: rgAsset): boolean {
               asset.market_hash_name.includes(keyword)
           );
 }
+
+export function getFadePercentage(asset: rgAsset, itemInfo: ItemInfo): number | undefined {
+    const FADE_TYPE_TO_CALCULATOR = {
```

Comment:
nit: move this outside of function

Reply:
Honestly, I'm not opposed to keeping this definition inside here since it'll only ever by used by this function.

Reply:
got it, a repetitive 3 field object declaration never hurt anyone 🤣